app/models/customer.rb

```diff
@@ -1,17 +1,19 @@
 class Customer < ApplicationRecord
   include PgSearch::Model
-  pg_search_scope :stemmed, ->(query, threshold) {
+  pg_search_scope :stemmed, lambda { |query, threshold|
     {
       query: query,
-      against: {name: "A", email: "B", company: "B", age: "B", status: "C"},
+      against: { name: 'A', email: 'B', bio: 'B', company: 'B', age: 'B', status: 'C' },
       using: {
-        tsearch: {prefix: true},
-        trigram: {threshold: threshold, only: [:name]}
+        tsearch: { prefix: true,
+                   tsvector_column: 'tsv' },
+        trigram: { threshold: threshold, only: [:name] }
       }
     }
   }
+
   scope :search_for, ->(query, threshold) { stemmed(query, threshold) if query.present? }
   scope :with_status, ->(status) { where(status: status) if status.present? }
-  scope :only_lawyers, ->(lawyers) { where("company like ?", "% and %") if lawyers }
+  scope :only_lawyers, ->(lawyers) { where('company like ?', '% and %') if lawyers }
```

Comment:
I'm actually surprised that single-quotes would even work here, since only double-quotes have string interpolation. Did you test it?